hack/lib/mutil-arch.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
delete this script

Reply:
No need since we can leave the buildx download/config to developer.

Reply:
we need to have buildx multiple platforms features config, leave the buildx download to developer.